TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/Drivebase.java

```diff
@@ -0,0 +1,275 @@
+package org.firstinspires.ftc.teamcode.hardware;
+
+import static org.firstinspires.ftc.teamcode.utility.Config.ENCODER_PER_INCH;
+
+import androidx.annotation.Nullable;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+import com.qualcomm.robotcore.hardware.IMU;
+
+import org.firstinspires.ftc.robotcore.external.Supplier;
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.teamcode.utility.Api;
+
+/**
+ * Class that contains the entire drivebase for the robot. Contains methods to move the robot both in auto and in teleop.
+ */
+public class Drivebase {
+    private final DcMotor frdrive;
```

Comment:
Comment a header block to distinguish that you're defining your private file-scope variables here.